jigglypuffRL/deeprl/ddpg/ddpg.py

```diff
@@ -172,7 +174,7 @@ def select_action(self, state, deterministic=False):
 
         return np.clip(
             action,
-            -self.env.action_space.high[0],
+            self.env.action_space.low[0],
```

Comment:
Wow

Reply:
This was not the reason it wasn't working though. It was because of a gradient being added to the loss I think

Reply:
Oh, was this also there in TD3? Because I'd used DDPG as a template for it.

Reply:
It wasn't there in TD3. Maybe DDPG stopped working after you added TD3

Reply:
TD3 also shouldn't have worked then?

Reply:
> It wasn't there in TD3. Maybe DDPG stopped working after you added TD3

Sorry, I meant maybe Ddpg stopped working in some other commit after td3 was made already. Like one of those refactors

Reply:
Cool